makei

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+# Licensed Materials - Property of IBM
+# 57XX-XXX
+# (c) Copyright IBM Corp. 2021
+
+# @author Tongkun
+#
+# Usage: makei [target]
+# 
+# Wrapper around `make` command so that runs make in every subdirectory if possible.
+# If enableLog is true, its output can be captured and logged into
+# a log directory.  The directory name is gleaned by scraping the output of IBMiMake.
+#
+# A named pipe is created, and a reader process (`tee`) is started in the background.  Make's
+# output is redirected to the pipe, where the reader writes it to both stdout and a log file.
+# The complexity is needed because merely using `tee` loses Make's exit code.  An alternate
+# solution, redirecting Make's output to a file and later sending it to stdout.
+#
+
+# enableLog=true
+enableLog=false
+exitCode=0
+
+# Create a named pipe that stdout and stderr will be funneled through, and start a tee reader process.
+pid=$$
+pipe="/tmp/pipe$pid"
+log="/tmp/makelog${pid}.log"
+mkfifo "$pipe"
+tee "$log" <"$pipe" &
+
+makeOpts=$*
+srcDir="$(pwd)"
+
+# Get the sourceCcsid from a given .ibmi.json. jq is prefered as the json parser
```

Comment:
if jq not found, awk is used and assumes that the sourceCssid attribute are on the same line